src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {

```

Comment:
**Use comments for this class to change all your implementations.**

Reply:
I did not understand. 
Does it means i need to comment constructors, methods like i did on the screenshot: 
![image](https://user-images.githubusercontent.com/105455921/197624618-843465f9-9213-4c8d-8a26-0fdf2926c1b6.png)
?

Reply:
Yes. And remove all commented code (constructor).